st4/__init__.py

```diff
@@ -116,6 +121,16 @@ def m_metals_executeClientCommand(self, params: Any) -> None:
                         location_to_encoded_filename(args[0]),
                         flags=sublime.ENCODED_POSITION
                     )
+        if params.get('command') == 'metals-show-stacktrace':
+            args = params.get('arguments')
+            def _replace_link(match) -> str:
+                url = urlparse(match.group(1))
+                command_args = json.loads(unquote(url.query))
+                path = urlparse(command_args[0]).path
+                location = command_args[1] + 1
+                return "href='subl:open_file_encoded {{\"file\": \"{}:{}\"}}'".format(path, location)
```

Comment:
We can merge as is and maybe update the support in Metals later (with sublime/vscode specific commands)

Reply:
Yes I will create an issue on Metals and make a PR for specific commands